src/components/Votes.jsx

```diff
@@ -0,0 +1,54 @@
+import { useEffect, useState } from "react"
+import { setVotes } from "../utils/api-requests"
+
+
+function Votes({votes, id, articlesOrComments}) {
+
+    const[currentVotes, setCurrentVotes] = useState(votes)
```

Comment:
votes is already held in state,  when you get your articles/article. So when that state changes, this whole ocmponent will automatically rerender. Therefore this state is not needed and leads to duplication of stored data.